.travis.yml

```diff
@@ -16,17 +16,18 @@ env:
   - ENCRYPTION_LABEL: "898e0934cf65"
   - COMMIT_AUTHOR_EMAIL: "janos.sallai@vanderbilt.edu"
 
+
+
 install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install md5sha1sum; fi
   - source devtools/travis-ci/install.sh
-  - conda config --add channels omnia
-  - conda config --add channels janschulz
-  - conda config --add channels bioconda
-  - conda config --add channels glotzer
-  - conda config --add channels mosdef
-
-script:
-  - conda build --python $PYTHON_VERSION devtools/conda-recipe
+  - conda config --set always_yes yes --set changeps1 no
+  - conda env create -q -n test-environment python=$PYTHON_VERSION -f environment.yml
+  - source activate test-environment
+  - pip install -e .
+  - py.test -v --cov=mbuild  --cov-report= --pyargs mbuild
+  - coveralls
 
 after_success:
+  - conda build --python $PYTHON_VERSION devtools/conda-recipe
```

Comment:
I might have misunderstood the appveyor idea, I thought appveyor was just a windows CI thing.

The conda build step doesn't run any tests but will be necessary if you want to automate conda package upload if branch==master someday. If we don't actually publish/upload anything to conda with travis, then this step is not required.